susemanager-utils/susemanager-sls/salt/packages/redhatproductinfo.sls

```diff
@@ -18,7 +18,7 @@ oraclerelease:
 amazonrelease:
   cmd.run:
     - name: cat /etc/system-release
-    - onlyif: test -f /etc/system-release -a ! -L /etc/system-release
+    - onlyif: test -f /etc/system-release && grep -i Amazon /etc/system-release >/dev/null
```

Comment:
Nit: I think the `-q` (quiet) switch has the same effect as piping grep into `/dev/null` and is a bit nicer: 
```bash
grep -iq Amazon /etc/system-release
```

Reply:
changed